src/hooks/useMultiplayerState.ts

```diff
@@ -194,6 +196,7 @@ export function useMultiplayerState({
   const onMount = useCallback(
     (app: TldrawApp) => {
       app.loadRoom(roomId);
+      app.document.name = `room-${roomId}`;
```

Comment:
I mentioned the room knowlege already in the other PR.

Reply:
fixed in the other PR, BC-6425